provisioner/terraform/resources.go

```diff
@@ -27,12 +25,14 @@ type agentAttributes struct {
 
 // A mapping of attributes on the "coder_app" resource.
 type agentAppAttributes struct {
-	AgentID      string                     `mapstructure:"agent_id"`
-	Name         string                     `mapstructure:"name"`
-	Icon         string                     `mapstructure:"icon"`
-	URL          string                     `mapstructure:"url"`
-	Command      string                     `mapstructure:"command"`
-	RelativePath bool                       `mapstructure:"relative_path"`
+	AgentID   string `mapstructure:"agent_id"`
+	Name      string `mapstructure:"name"`
+	Icon      string `mapstructure:"icon"`
+	URL       string `mapstructure:"url"`
+	Command   string `mapstructure:"command"`
+	Subdomain bool   `mapstructure:"subdomain"`
+	// RelativePath is deprecated in favor of Subdomain.
+	RelativePath *bool                      `mapstructure:"relative_path"`
```

Comment:
I don't think this needs to be a pointer anymore

Reply:
We prefer the deprecated value over the new value if it's set, so we have to use a pointer for this to determine if it was explicitly set or not. I will add a comment